config.toml

```diff
@@ -356,6 +356,10 @@ logo = "pro-logo.svg"
 url = "https://www.purefacts.com"
 logo = "purefacts.png"
 
+[[params.qonto]]
```

Comment:
This will need to be aligned
```suggestion
[[params.users]]
```

Reply:
Sorry, too tired. I fixed and rebased. Thank you.